script/Ciel/Day 14/14_01_CIEL14_1.txt

```diff
@@ -1,452 +1,501 @@
 [sha:26c5fd5faac96c37dcdccd7e00ab17584840b5b4]{
 -- Page 1, Offset 37003.
 -- 今でも私は、この感情を保管している。
-Even now, I hold onto these feelings.
+Even now, I keep these feelings close to my heart.
 }
 [sha:5c9a65d86cb613adac73d72838e7d1c290dcd23b]{
 -- Page 1, Offset 37004.
 -- もう利用価値もなく、共感もできないかつての<記録|じぶん>。
-My former self, no longer having value, unable to feel even sympathy.
+Past <memories|s e l v e s   > I no longer have any use for;
+ that no longer resonate with me.
 }
 [sha:f225dce7e0febc44a2f460f13c2be16050a83f4b]{
 -- Page 1, Offset 37005.
 -- 無限の航路に身を投じる前に抱いた、
-I still carry the memories of what I used to be,
+The final moments of the man I used to be―――
 }
 [sha:043a0a685797a575a32f58abbb530656f99484a0]{
 -- Page 1, Offset 37006.
 -- 私だったものの<断末魔|おもいで>を。
-before I threw myself into the endless course.
+Before I threw myself into my endless voyage.
 }
 [sha:60ac162a0aaab69effbbcd4b30469f240eda48d9]{
 -- Page 2, Offset 37007.
 -- 目的は変わったのか。
-Perhaps my purpose has changed.
+I wonder if my purpose has changed.
 }
 [sha:1371498bc2cfc2914825f35ac03ce6f3078bcc6a]{
 -- Page 2, Offset 37008.
 -- いや、到達点そのものは変わっていない。
-No, the very place I arrive at still remains the same.
+No; my destination itself remains the same.
 }
 [sha:1a153e2e6b5523df884515af226664f772feaef7]{
 -- Page 2, Offset 37009.
 -- 私は永遠を目指す。
-I aim toward eternity.
+I set my sights on eternity.
 }
 [sha:c9dcd6761a66a055107e694f997291ce58a5345d]{
 -- Page 2, Offset 37010.
 -- 一切の報酬もなく永遠を目指そうとした。
-I aimed toward an eternity without any reward in sight.
+I set my sights on eternity even though I stood to gain nothing from it.
 }
 [sha:9bd92bab7559c13f4ee039aceb6381d2f0c9dd7e]{
 -- Page 2, Offset 37011.
 -- それはなんて純粋で、<穢|けが>れのない意思だったのだろうか。
-What a pure and innocent purpose that was.
+Could there be any other purpose as pure and immaculate as that?
 }
 [sha:631bdb462d260188dcb0956431758e837b283ab7]{
 -- Page 3, Offset 37012.
 -- 変わってしまった。
-I've changed.
+But something has changed.
 }
 [sha:ddf8d25c79a0b4905f479594ff34faee59ae8501]{
 -- Page 3, Offset 37013.
 -- 私は、変わってしまった。
-I've definitely changed.
+%{i}I%{/i} have changed.
 }
 [sha:329365cdd5280bf6eaae7f73dde66855b6896d1e]{
 -- Page 3, Offset 37014.
 -- 永遠の定義が、それを証明する方法が変わってしまった。
-The way I determined the definition of eternity has changed.
+My definition of eternity has changed,
+ along with the method by which I would prove it.
 }
 [sha:d465a9f225c09da68389e467e5807c23e8bb27a9]{
 -- Page 3, Offset 37015.
 -- 私は長い時間を経て、完成を見たかったのに。
-Even though I wanted to see it complete after all this time.
+// 完成 is EXTREMELY abstract here.
+I have spent many eons aspiring to witness completion.
 }
 [sha:444175de01f842e3702ae8bd845cbc1f8adc595d]{
 -- Page 3, Offset 37016.
 -- 今は完成を得る為に、長い時間を求めている。
-Now I wish for an even longer time in order to complete it.
+Yet now, I wish for more eons in order to %{i}obtain%{/i} completion.
```

Comment:
Looking at this again, I think this might be slightly wrong.
He hasn't 

>I had aspired to go through many eons in order to witness completion.

>Yet now, I desire those many eons in order to %{i}obtain%{/i} completion.